rocketc/static/html/rocketc.html

```diff
@@ -11,6 +11,10 @@
                 {{response}}
             {% endif %}
 
+        {% elif not team_view %}
+
+            <strong><p>You need to join a team before this will work. Go to the <a href='{% url "teams_dashboard" user_data.course_id %}'>teams page</a> to select a team to join or get in <a href='{% url "support:contact_us" %}'>touch with support.</a></p></strong>
```

Comment:
To maintain code style, let's use `href="{% url 'teams_dashboard' user_data.course_id %}"` and `href="{% url 'support:contact_us' %}"`

Reply:
Done